source/sessions/driver-sessions.rst

```diff
@@ -1009,6 +1043,8 @@ ensure that they close any explicit client sessions and any unexhausted cursors.
     * Iterate through enough documents (3) to force a ``getMore``
     * Assert that the server receives a non-zero lsid equal to the lsid that ``find`` sent.
 
+11. For a command to a mongos that includes a readPreference, verify that the ``lsid`` on query commands is added inside the ``$query`` field, and NOT as a top-level field
```

Comment:
Mention that this is only needed if the driver has not implemented OP_MSG? ``$query`` is only used with OP_QUERY.

Reply:
👍 Fixed